api/grpcserver/v2alpha1/activation.go

```diff
@@ -245,17 +246,17 @@ func toAtxOperations(filter *spacemeshv2alpha1.ActivationRequest) (builder.Opera
 	if filter == nil {
 		return ops, nil
 	}
-	if filter.NodeId != nil {
+	if filter.SmesherId != nil {
```

Comment:
Consider 
```suggestion
	if len(filter.SmesherId) > 0 {
```

btw, the field could have a plural name to make it clearer it's a list.